runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/InfoBoxManager.java

```diff
@@ -71,6 +72,14 @@ public void addInfoBox(InfoBox infoBox)
 		updateInfoBoxImage(infoBox);
 		infoBoxes.add(infoBox);
 		refreshInfoBoxes();
+
+		BufferedImage image = infoBox.getImage();
+
+		if (image instanceof AsyncBufferedImage)
```

Comment:
Did you give any thought to making the spritemanager also use asyncbufferedimage? It might simplify a lot of 48f6471148c15cc9e1d63d685acbf3948ac62cb2

Reply:
I agree, this sounds like good idea. And also would prevent the need of exposing methods from infoboxManager because then it would be handled in https://github.com/runelite/runelite/pull/9275/commits/0c32d787dc652ef6edbc95c3bf89d03113c1bffb

Reply:
Its impossible to use AsyncBufferedImage here because I don't know the size of the image in advance, and AsyncBufferedImage cannot change size